wls-gui-wahllokalsystem/tests/stores/ereignisStore.spec.ts

```diff
@@ -252,6 +471,184 @@ describe("ereignisStore.ts", () => {
       expect(eintragToChange.uhrzeit).toBeUndefined();
     });
   });
+
+  describe("watch", () => {
+    describe("schliessungsUhrzeitSent", () => {
+      it("should_updateEreignisart_when_schliessungsuhrzeitSentHasChanged", async () => {
+        const schliessungsuhrzeitSend = new Date();
+
+        const ereignisEintraege = [
+          createEreignis(),
+          createEreignis(),
+          createEreignis(),
+        ];
+        unitUnderTest.wahlbezirkEreignisse.ereigniseintraege =
+          ereignisEintraege;
+
+        wahlbezirkStore.schliessungsUhrzeitSent = schliessungsuhrzeitSend;
+
+        const spyGetEreignisArtForDateRelatedToSchliessungsuhrzeit = spyOn(
+          ImportAllFromEreignisArt,
+          "getEreignisArtForDateRelatedToSchliessungsuhrzeit"
+        );
+
+        await nextTick(); //wait till store has processed changed schliessungsuhrzeit
+        await nextTick(); //wait till
```

Comment:
entweder das zweite `nextTick()` entfernen oder den kommentar vervollständigen
```suggestion
        await nextTick(); // wait until store has processed change of schliessungsuhrzeit
```

Reply:
fixed mit b1c8e06f0a50b731ec5e9a79f687d8eebf9d839b